### PR TITLE
Configurable output settings

### LINKED ISSUE #16: Output settings
Each exercise should have an output configuration section, that defines what is displayed on the output. I imagine something like the following:

```yml
output:
  records: false
  flags: true
  special_records: true
  memory:
    from: '0001'
    to: 'A03F'
```

Also, each field should have a default, that will be used if the key is not present:

* `records: true`
* `flags: false`
* `special_records: false`
* `memory: false` (or any null-like value that means "it should not be displayed")

The visualization for the special records and the memory should be similar to the record, but using multiple columns for the memory:
image

### PR DESCRIPTION
**Pull request by Charlyzzz**
Closes #16

### CODE REVIEW

File: lib/extension/string_extension.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module StringExtension
+  refine String do
```
Comment (flbulgarelli) [inline lib/extension/string_extension.rb]: I'd put this in a plain-old open class.

**Comment by faloi** [comment]
@Charlyzzz can you provide a few screenshots with sample outputs?

**Comment by Charlyzzz** [comment]
This output settings produces
~~~ruby
memory: { from: '0007', to: '000B' },
flags: true,
records: true,
special_records: true
~~~
(The screen is zoomed out for obvious reasons)
image

**Comment by faloi** [comment]
Two things I imagined a bit different:

* The memory could grow to the right, up to 10? cells and then use a second row if necessary. The way you did it is pretty inconvenient for larger ranges.
* Given that the rest of the tables are small and fixed, they can share the same row whenever it's possible. That way, we reduce the output's height and less scrolling is necessary.

**Comment by Charlyzzz** [comment]
There is a method inside the renderer that determines for each kind of output how many elements each column has. If there are less than 7, it is that number. Otherwise, it takes 4.

Let me play with that

**Comment by Charlyzzz** [comment]
Like this?
image

**Comment by flbulgarelli** [comment]
And what about like this?

| C | N | V | Z
|-----|------|-----|----
| 1 | 0  | 1 | 1

**Comment by Charlyzzz** [comment]
V1
image

V2 (as @flbulgarelli suggested)
image

@faloi

**Comment by Charlyzzz** [comment]
Please, DONT MERGE!
I won't push the last version until the layout is OK.

**Comment by Charlyzzz** [comment]
This is the output of the version i'm pushing
image

**Pull request merged**